cassandra_nodetool/datadog_checks/cassandra_nodetool/data/conf.yaml.example

```diff
@@ -34,13 +34,18 @@ instances:
 
   ## @param password - string - optional
   ## Username from the credentials needed to connect to the host. These are the credentials for the JMX server.
-  ## For the check to work, this user must have a read/write access so that the Datadog Cassandra Nodetool check can execute the `status` command 
+  ## For the check to work, this user must have a read/write access so that the Datadog Cassandra Nodetool check can execute the `status` command
   #
   #  password: <PASSWORD>
 
-  ## @param tags  - list of key:value element - optional 
+  ## @param ssl - boolean - optional
```

Comment:
The default value should be defined here.

`## @param ssl - boolean - optional - default:false`